### ISSUE TITLE
broken package

### ISSUE THREAD

**Issue by forostm** [KaOS-Community-Packages/digue-git#1]
Github source does not exist (error 404)

**Comment by bvaudour** [comment]
Indeed. No idea if it has moved otherwhere. So marked as removal candidate.

**Comment by bvaudour** [comment]
Correction: it is currently available on pypi. Only thing to do is to launch "sudo pip3 install digue" to launch it.

As digue isn’t a depend of any KaOS package no reason to keep it. It will be removed.

**Comment by forostm** [comment]
Yes, I agree.
